os/fs/littlefs/littlefs/lfs.h

```diff
@@ -104,6 +104,7 @@ enum lfs_error {
     LFS_ERR_NOMEM       = -12,  // No more memory available
     LFS_ERR_NOATTR      = -61,  // No data/attr available
     LFS_ERR_NAMETOOLONG = -36,  // File name too long
+	LFS_ERR_SIG = -10,		// Signiture crpt__fs
```

Comment:
so mount is failed because super block has corrupted. then I think LFS_ERR_CORRUPT is enough to handle it.